src/ptvsd/common/attach_script_ptvsd_pid.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I was thinking about this and this whole code is pretty much a copy of `pydevd_attach_to_process/attach_pydevd.py` and `pydevd_attach_to_process/attach_script.py`... if multiprocessing will work with the plain `pydevd` attach later on, maybe this code shouldn't be needed (we may need to have multiprocessing done first so that this works though).

i.e.: it could use `pydevd_attach_to_process/attach_pydevd.main()` just passing the proper `port` and `pid` to connect to an adapter that'd be waiting for the connection and then do the same handshake that `multiprocessing` will do (right now it doesn't have the mode to wait for an incoming connection, but that should be easy to do).

We don't necessarily need to do this now, but it may be worth doing it in the future so that we don't have to change both places if some change is needed.

Reply:
Ah, so we'd use `evaluate` request to load ptvsd into the process here after pydevd does its own injection and connects?

Reply:
Yes ;)